packages/ordered-list/README.md

```diff
@@ -0,0 +1,32 @@
+# Ordered List
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/ordered-list.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/ordered-list/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/ordered-list
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/ordered-list
+```
+
```

Comment:
Would it be helpful to add a usage example here?

Reply:
Done